### PR TITLE
Audit module 0.15.1

### PR DESCRIPTION
**Pull request by koenderks**
This PR updates most UI's to a more JASPy and more user friendly style. The files containing almost all relevant changes are `auditCommonFunction.R` and `auditCommonFunctionBayesian.R`

Example, before:
image

After:
image

Closes https://github.com/jasp-stats/INTERNAL-jasp/issues/1531
Closes https://github.com/jasp-stats/jaspAudit/issues/21
Closes https://github.com/jasp-stats/INTERNAL-jasp/issues/1521
Closes https://github.com/jasp-stats/INTERNAL-jasp/issues/1522
Closes https://github.com/jasp-stats/jaspAudit/issues/104
Closes https://github.com/jasp-stats/jaspAudit/issues/60

Also replaces `themeJasp()` with `geom_rangeframe() + themeJaspRaw()`

### CODE REVIEW

**Comment by vandenman** [comment]
@koenderks did you mean to assign reviewers already?

**Comment by Kucharssim** [comment]
> did you mean to assign reviewers already?

Koen asked me originally, but I am happy if @FBartos review it instead!

**Comment by FBartos** [comment]
well, I have 4 PRs to manage, so if you were planning to review it @Kucharssim it would be very appreciated :)

**Comment by koenderks** [comment]
@vandenman @Kucharssim Yes, this is ready for review.

**Review by vandenman — CHANGES_REQUESTED** [review]
You changed quite a bunch of option names here. To guarantee backwards compatibility, you should also update Upgrader.qml.

I couldn't really find any bugs, but there are simply too many changes to carefully review all the code.

**Comment by koenderks** [comment]
Some options have been removed (like number of items in planning), how can we indicate this in the upgrader?

File: R/auditCommonFunctions.R [hunk 1]
```diff
@@ -886,79 +750,56 @@
...
+      if (options[["materiality_type"]] == "materiality_abs") {
+        expTMP <- expTMP / parentOptions[["N.units"]]
+      }
+      if (expTMP != 0 && expTMP < 1 && expTMP >= parentOptions[["materiality_val"]] && !options[["min_precision_test"]]) {
         # Error if the expected errors exceed the performance materiality
-        parentContainer$setError(gettext("Analysis not possible: Your expected errors are higher than the performance materiality."))
+        parentContainer$setError(gettext("Analysis not possible: The expected errors are higher than the performance materiality."))
         return(TRUE)
       }
       if (.jfaAuditRiskModelCalculation(options) >= 1) {
```
Comment (AlexanderLyNL) [inline R/auditCommonFunctions.R:788]: Why is this compared to 1?

File: R/auditCommonFunctions.R [hunk 2]
```diff
@@ -1580,682 +1406,629 @@
...
+    }
+
+    if (options[["bayesian"]]) {
+      dist <- switch(options[["likelihood"]],
+        "binomial" = "Beta",
+        "poisson" = "Gamma",
+        "hypergeometric" = "Beta-binomial"
+      )
+    } else {
+      dist <- switch(options[["likelihood"]],
```
Comment (AlexanderLyNL) [inline R/auditCommonFunctions.R:1768]: distName (?) Here you're creating the name of the distribution to be used in the plot title.

File: R/auditCommonFunctions.R [hunk 3]
```diff
@@ -1580,682 +1406,629 @@
...
-        
+
         plot <- ggplot2::ggplot(data = dPlot, mapping = ggplot2::aes(x = x, y = y, fill = type)) +
           ggplot2::geom_bar(stat = "identity", col = "black", size = 1) +
-          ggplot2::scale_y_continuous(name = gettext("Required sample size (n)"), breaks = yBreaks, limits = yLimits) +
+          ggplot2::scale_y_continuous(name = gettext("Minimum sample size"), breaks = yBreaks, limits = yLimits) +
           ggplot2::coord_flip() +
-          ggplot2::annotate("text", y = k, x = c(3, 2, 1), label = k, size = 6, vjust = 0.5, hjust = -0.25) +
-          ggplot2::annotate("text", y = n, x = c(3, 2, 1), label = n, size = 6, vjust = 0.5, hjust = -0.75) +
+          ggplot2::annotate("text", y = k, x = length(n):1, label = k, size = 6, vjust = 0.5, hjust = -0.25) +
```
Comment (AlexanderLyNL) [inline R/auditCommonFunctions.R:1847]: Why is the x-axis flipped? Going from n to 1?

File: R/auditCommonFunctions.R [hunk 4]
```diff
@@ -2724,909 +2420,718 @@
...
-      evaluationState 	<- evaluationContainer[["evaluationState"]]$object
-      errorLabel 		<- evaluationState[["k"]]
-      
-      if (options[["display"]] == "displayNumbers") {
-        if (options[["estimator"]] %in% c("directBound", "differenceBound", "ratioBound", "regressionBound")) {
-          boundLabel <- round(evaluationState[["upperBound"]] / planningOptions[["populationSize"]], 3)
-          mleLabel <- round(evaluationState[["mle"]] / planningOptions[["populationSize"]], 3)
-          precisionLabel <- round(evaluationState[["precision"]] / planningOptions[["populationSize"]], 3)
+      state <- evaluationContainer[["evaluationState"]]$object
+      errors <- state[["x"]]
```
Comment (AlexanderLyNL) [inline R/auditCommonFunctions.R:2520]: The naming "x" is rather elusive.

File: R/auditCommonFunctions.R [hunk 5]
```diff
@@ -2724,909 +2420,718 @@
...
+      if (options[["display"]] == "number") {
+        if (options[["method"]] %in% c("direct", "difference", "quotient", "regression")) {
+          boundLabel <- round(state[["ub"]], 3)
+          mleLabel <- round(state[["mle"]], 3)
+          precisionLabel <- round(state[["precision"]], 3)
         } else {
-          boundLabel <- round(evaluationState[["confBound"]], 3)
-          mleLabel <- round(evaluationState[["mle"]], 3)
-          precisionLabel <- round(evaluationState[["precision"]], 3)
+          boundLabel <- round(state[["ub"]], 3)
```
Comment (AlexanderLyNL) [inline R/auditCommonFunctions.R:2528]: isn't upperBound more clear?

File: R/auditCommonFunctions.R [hunk 6]
```diff
@@ -3966,186 +3322,205 @@
...
+  adjustedMateriality <- (prevOptions[["materiality_val"]] / (1 - (sum(sample[[options[["values"]]]]) / prevOptions[["N.units"]])))
+  alphaPosterior <- prior[["description"]]$alpha + t
+  betaPosterior <- prior[["description"]]$beta + n - t
+  posterior <- list(
+    description = list(alpha = alphaPosterior, beta = betaPosterior),
+    statistics = list(
+      median = qbeta(0.5, alphaPosterior, betaPosterior),
+      mean = alphaPosterior / (alphaPosterior + betaPosterior),
+      mode = (alphaPosterior - 1) / (alphaPosterior + betaPosterior - 2),
+      ub = qbeta(options[["conf_level"]], alphaPosterior, betaPosterior),
```
Comment (AlexanderLyNL) [inline R/auditCommonFunctions.R:3491]: ciUpper

File: R/auditCommonFunctionsBayesian.R [hunk 7]
```diff
@@ -14,746 +14,415 @@
...
-                            "median" 		= gettextf("The implicit sample is derived from equal prior probabilities: <i>p(H%1$s) = p(H%2$s) = 0.5</i>", "\u208B", "\u208A"),
-                            "hypotheses" 	= gettextf("The implicit sample is derived from custom prior probabilities: <i>p(H%1$s) = %2$s, p(H%3$s) = %4$s</i>", "\u208B", options[["pHmin"]], "\u208A", 1 - options[["pHmin"]]),
-                            "sample" 		= gettextf("The implicit sample is derived from an earlier sample of %1$s observations containing %2$s errors.", options[["sampleN"]], options[["sampleK"]]),
-                            "factor" 		= gettextf("The implicit sample is derived from an earlier sample of %1$s observations containing %2$s errors weighted by a factor <i>f = %3$s</i>.", options[["sampleN"]], options[["sampleK"]], options[["factor"]]))
+    table$dependOn(options = c("tableImplicitSample", "likelihood", "tablePrior"))
+
+    table$addColumnInfo(name = "n", title = gettext("Equivalent sample size"), type = "number")
+    table$addColumnInfo(name = "x", title = gettext("Equivalent errors"), type = "number")
```
Comment (AlexanderLyNL) [inline R/auditCommonFunctionsBayesian.R:79]: Why is this named "x"?

File: R/auditCommonFunctionsBayesian.R [hunk 8]
```diff
@@ -14,746 +14,415 @@
...
-                    unseenLowerBound = lowerBound, unseenUpperBound = upperBound)
-    return(results)
+
+  conf_lb <- (1 - parentState[["conf.level"]]) / 2
+  conf_ub <- parentState[["conf.level"]] + conf_lb
+  alpha <- parentState[["posterior"]][["description"]]$alpha
+  beta <- parentState[["posterior"]][["description"]]$beta
+
+  if (parentState[["method"]] == "poisson") {
+    lb <- qgamma(conf_lb, shape = alpha, rate = beta)
```
Comment (AlexanderLyNL) [inline R/auditCommonFunctionsBayesian.R:39]: ciLower

**Pull request merged**